generate/generate_nircam.py

```diff
@@ -318,100 +342,126 @@
 aperture_collection = pysiaf.ApertureCollection(aperture_dict)
 
 emulate_delivery = True
-emulate_delivery = False
+# emulate_delivery = False
 
 if emulate_delivery:
     pre_delivery_dir = os.path.join(JWST_DELIVERY_DATA_ROOT, instrument)
     if not os.path.isdir(pre_delivery_dir):
         os.makedirs(pre_delivery_dir)
 
     # write the SIAF files to disk
-    filenames = pysiaf.iando.write.write_jwst_siaf(aperture_collection, basepath=pre_delivery_dir, file_format=['xml', 'xlsx'])
+    filenames = pysiaf.iando.write.write_jwst_siaf(aperture_collection, basepath=pre_delivery_dir, file_format=['xml'], label='jwstsiaf-129')
 
-    pre_delivery_siaf = pysiaf.Siaf(instrument, basepath=pre_delivery_dir)
+    # 1/0
```

Comment:
Delete commented out code?

Reply:
done. 
I cleaned this part up in the next commit. I will resolve the applicable comments.